CONTRIBUTING.md

```diff
@@ -68,6 +68,22 @@ Key scripts include:
 Ensure you run `lint` and `fmt` before pushing.
 The [CI](https://github.com/wisdom-oss/frontend/actions) enforces this.
 
+## Git Pre-Commit Hooks
+
+In order to automatically format and lint the codebase, you can use husky (https://typicode.github.io/husky/get-started.html)
+install via
+'''sh
+npm install --save-dev husky
+'''
+
+and make sure npm still functions with repeating
+'''sh
+npm install
+'''
```

Comment:
use back-ticks for [code fences](https://github.github.com/gfm/#fenced-code-blocks)